roles/download/defaults/main.yml

```diff
@@ -386,9 +386,15 @@ calicoctl_binary_checksums:
     v3.15.2: 49165f9e4ad55402248b578310fcf68a57363f54e66be04ac24be9714899b4d5
 
 helm_archive_checksums:
-  arm: ca8792da269b72235987ea7245d1450a859b2c0658f591737d74b6c56cd9b1fa
-  amd64: 3fff0354d5fba4c73ebd5db59a59db72f8a5bbe1117a0b355b0c2983e98db95b
-  arm64: 87811b648ed9f4c84d3cb67bbea9b666bb7f6dd0ff6aca148b65f91058f73953
+  arm:
+    v3.5.0v: ca8792da269b72235987ea7245d1450a859b2c0658f591737d74b6c56cd9b1fa
```

Comment:
need to remove `v` after `v3.5.0`..

Reply:
@oomichi I'm so sorry, this PR is so small. How can I do so many mistakes ! :see_no_evil: